console/account/how-to/open-a-support-ticket.mdx

```diff
@@ -7,36 +7,33 @@ content:
   paragraph: This page explains how to open a support ticket
 tags: support help ticket support-ticket assistance
 dates:
-  validation: 2024-01-02
+  validation: 2024-01-17
   posted: 2021-06-22
 categories:
   - console
 ---
 
-Scaleway provides 24/7 basic technical support by ticket for free. You can contact our support team directly from the console. You can [upgrade your support level](/console/account/how-to/configure-support-plans/) to benefit from phone support and a dedicated account manager.
+Scaleway provides 24/7 basic technical support by ticket for free. You can contact our support team directly from the console. You can also [upgrade your support level](/console/account/how-to/configure-support-plans/) to benefit from phone support and a dedicated account manager.
 
 <Macro id="iam-requirements" />
 
 <Message type="requirement">
   - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
 </Message>
 
-If you have any inquiries and wish to speak to our support team, you may open a support ticket by clicking on **Support** in the lower left menu.
-
-<Lightbox src="scaleway-support-in-the-menu.webp" alt="" />
-
-Alternatively, you can click <Icon name="assistance" /> in the top menu. The [support center](https://console.scaleway.com/support/tickets) displays.
-
-<Lightbox src="scaleway-support-center.webp" alt="" />
-
-Click **+ Open a new ticket** to do so.
+If you have any inquiries and wish to speak to our support team:
+1. Click on **Support** in the lower left menu. The [support center](https://console.scaleway.com/support/tickets) displays.
+2. Click the **+ Create a ticket** button. 
+3. Fill in the required information and click **Submit ticket**.
 
+<Message type="note">
+Alternatively, you can click <Icon name="assistance" /> in the top menu. 
```

Comment:
I think this has been removed, I can't see it on the console anymore.

Reply:
Still in AB test, the new version will be deployed by the end of the week, we could replace the icon by "Help button" and wait to merge the PR